src/components/Text/Text.stories.tsx

```diff
@@ -1,18 +1,141 @@
-import { Story, Meta } from '@storybook/react';
+import type { StoryObj } from '@storybook/react';
+import clsx from 'clsx';
 import React from 'react';
 
-import { Text, Props } from './Text';
+import Text from './Text';
+import type { Color } from './Text';
+import styles from './Text.stories.module.css';
 
 export default {
-  title: 'Atoms/Text/Text',
```

Comment:
nit: do we wanna keep this organization?

Reply:
Aye aye good catch, will change